mathics/builtin/specialfns/othogonal.py

```diff
@@ -1,294 +0,0 @@
-"""
```

Comment:
Why is this removed? It is listed in https://reference.wolfram.com/language/guide/SpecialFunctions.html

Reply:
Actually, was renamed to *orthogonal*